enforcer-rules/src/test/java/com/google/cloud/tools/opensource/enforcer/LinkageCheckerRuleTest.java

```diff
@@ -150,11 +155,42 @@ public void testExecute_shouldFailForBadProject() throws RepositoryException {
           "The rule should raise an EnforcerRuleException for artifacts missing dependencies");
     } catch (EnforcerRuleException ex) {
       // pass
+      verify(mockLog).error(ArgumentMatchers.startsWith("Linkage Checker rule found errors."));
       Assert.assertEquals(
           "Failed while checking class path. See above error report.", ex.getMessage());
     }
   }
 
+  @Test
+  public void testExecute_shouldFailForBadProject_reachableErrors() throws RepositoryException {
+    try {
+      // This pair of artifacts contains classes missing reachable errors on Verify.verify
```

Comment:
what does "classes missing reachable errors" mean? Consider rephrasing

Reply:
Updated to

      // This pair of artifacts contains linkage errors on grpc-core's use of Verify. Because
      // grpc-core is included in entry point jars, the errors are reachable.